galasa-parent/gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.jvmargs=-Xmx4096M
 
-isRelease=false
+isMainOrRelease=false
```

Comment:
Might be helpful to indicate what this flag controls in a comment in as many places as possible.